slack.ts

```diff
@@ -0,0 +1,69 @@
+import * as querystring from "querystring";
+import fetch from "node-fetch";
+
+const tkn = process.env.SLACK_TOKEN;
+const domain = process.env.SLACK_DOMAIN;
+
+interface Config {
+    channels: string[];
+}
+
+async function sendMessage(message : string, config : any) : Promise<void> {
```

Comment:
You should rebase this branch on top of the current master so you can implement `GenericNotifier` (which is an interface common to all plugins).